cmake/OpenCVFindOpenEXR.cmake

```diff
@@ -33,7 +39,7 @@ SET(SEARCH_PATHS
 MACRO(FIND_OPENEXR_LIBRARY LIBRARY_NAME LIBRARY_SUFFIX)
     string(TOUPPER "${LIBRARY_NAME}" LIBRARY_NAME_UPPER)
     FIND_LIBRARY(OPENEXR_${LIBRARY_NAME_UPPER}_LIBRARY
-        NAMES ${LIBRARY_NAME}${LIBRARY_SUFFIX}
+        NAMES ${LIBRARY_NAME} ${LIBRARY_NAME}${LIBRARY_SUFFIX}
```

Comment:
This would re-introduce mess with names again.

All libraries for found components must have the same suffix. Different subsets must be forbidden.